### ISSUE TITLE
[Problem/Bug]: Multiple Browser Instances Sharing User Data Folder Cannot Have Different Settings

### ISSUE THREAD

**Issue by vrad-joshuamckelvey** [MicrosoftEdge/WebView2Feedback#4593]
### What happened?

WebView2 fails to launch when two WebView2 browser instances are sharing the same User Data Folder with differing CoreWebView2EnvironmentOptions.AllowSingleSignOnUsingOSPrimaryAccount values. 

The second browser throws the error below:
```
System.Runtime.InteropServices.COMException: 'The group or resource is not in the correct state to perform the requested operation. (Exception from HRESULT: 0x8007139F)'
```

### Importance

Important. My app's user experience is significantly compromised.

### Runtime Channel

Stable release (WebView2 Runtime)

### Runtime Version

125.0.2535.67

### SDK Version

1.0.2535.41

### Framework

Winforms

### Operating System

Windows 11

### OS Version

10.0.22631

### Repro steps

1. Launch WebView2 in dialog with  CoreWebView2EnvironmentOptions.AllowSingleSignOnUsingOSPrimaryAccount set to false.
L2. Launch WebView2 in another dialog with  CoreWebView2EnvironmentOptions.AllowSingleSignOnUsingOSPrimaryAccount set to true.

### Repros in Edge Browser

No, issue does not reproduce in the corresponding Edge version

### Regression

Regression in newer Runtime

### Last working version (if regression)

124.0.2478.109

**Comment by vrad-joshuamckelvey** [comment]
I can provide a demo application as well if that would help.

**Comment by vbryh-msft** [comment]
It should be by design - it is browser feature. If you are running wv2 with the same UDF they share browser process. Could you please double check that it is working in 124?

**Comment by vbryh-msft** [comment]
[About webview2 process model](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-model?tabs=csharp) - just to make sure we are on the same page.
Multiple profiles for the same UDF [are ok](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/multi-profile-support?tabs=dotnetcsharp).
Just a note, because server with shortcuts was mentioned -  Avoid storing user settings on a network drive. This can result in slowdowns, crashes, or loss of data [link](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/user-data-folder?tabs=win32#suitable-custom-udf-locations).
What are different command line arguments, which are needed?

**Comment by SpoilerJones** [comment]
(I work with @pushkin- on this project.)  Is the point that our single UDF (despite the separate profiles) means there's only one browser process, shared by multiple (otherwise separate) instances of our app?

**Comment by vbryh-msft** [comment]
yes - I have linked a [ doc ](https://learn.microsoft.com/en-us/microsoft-edge/webview2/concepts/process-model?tabs=csharp#webview2-runtime-processes-and-the-user-data-folder).
